src/System.Net.Security/src/System/Net/_SSPIWrapper.cs

```diff
@@ -0,0 +1,873 @@
+
+            return errorCode;
+        }
+
+        internal static int AcceptSecurityContext(SSPIInterface SecModule, ref SafeFreeCredentials credential, ref SafeDeleteContext context, Interop.Secur32.ContextFlags inFlags, Interop.Secur32.Endianness datarep, SecurityBuffer inputBuffer, SecurityBuffer outputBuffer, ref Interop.Secur32.ContextFlags outFlags)
+        {
+            if (Logging.On)
+            {
+                Logging.PrintInfo(Logging.Web,
+                    "AcceptSecurityContext(" +
+                    "credential = " + credential.ToString() + ", " +
+                    "context = " + Logging.ObjectToString(context) + ", " +
+                    "inFlags = " + inFlags + ")");
+            }
+
+            int errorCode = SecModule.AcceptSecurityContext(ref credential, ref context, inputBuffer, inFlags, datarep, outputBuffer, ref outFlags);
+
+            if (Logging.On) Logging.PrintInfo(Logging.Web, SR.Format(SR.net_log_sspi_security_context_input_buffer, "AcceptSecurityContext", (inputBuffer == null ? 0 : inputBuffer.size), outputBuffer.size, (Interop.SecurityStatus)errorCode));
+
+            return errorCode;
+        }
+
+        internal static int AcceptSecurityContext(SSPIInterface SecModule, SafeFreeCredentials credential, ref SafeDeleteContext context, Interop.Secur32.ContextFlags inFlags, Interop.Secur32.Endianness datarep, SecurityBuffer[] inputBuffers, SecurityBuffer outputBuffer, ref Interop.Secur32.ContextFlags outFlags)
+        {
+            if (Logging.On)
+            {
+                Logging.PrintInfo(Logging.Web,
+                    "AcceptSecurityContext(" +
+                    "credential = " + credential.ToString() + ", " +
+                    "context = " + Logging.ObjectToString(context) + ", " +
+                    "inFlags = " + inFlags + ")");
+            }
+
+            int errorCode = SecModule.AcceptSecurityContext(credential, ref context, inputBuffers, inFlags, datarep, outputBuffer, ref outFlags);
+
+            if (Logging.On) Logging.PrintInfo(Logging.Web, SR.Format(SR.net_log_sspi_security_context_input_buffers, "AcceptSecurityContext", (inputBuffers == null ? 0 : inputBuffers.Length), outputBuffer.size, (Interop.SecurityStatus)errorCode));
+
+            return errorCode;
+        }
+
+        internal static int CompleteAuthToken(SSPIInterface SecModule, ref SafeDeleteContext context, SecurityBuffer[] inputBuffers)
+        {
+            int errorCode = SecModule.CompleteAuthToken(ref context, inputBuffers);
+
+            if (Logging.On) Logging.PrintInfo(Logging.Web, SR.Format(SR.net_log_operation_returned_something, "CompleteAuthToken()", (Interop.SecurityStatus)errorCode));
+
+            return errorCode;
+        }
+
+        public static int QuerySecurityContextToken(SSPIInterface SecModule, SafeDeleteContext context, out SecurityContextTokenHandle token)
```

Comment:
About half of these methods use `SecMethod`, the other half `secMethod`.  The latter is definitely preferred.